org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/DOMDocTypeFormatter.java

```diff
@@ -115,6 +129,9 @@ private void formatDTDNodeDecl(DTDDeclNode nodeDecl, XMLFormattingConstraints pa
 			if (internalDecls == null) {
 				for (DTDDeclParameter parameter : attlist.getParameters()) {
 					replaceLeftSpacesWithOneSpace(parameter.getStart(), edits);
+					// replace current quote with preferred quote in the case:

```

Comment:
Could add please à sample in comment for the prévois line code with replaceLeftSpacesWithOneSpace...